letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -510,6 +575,23 @@ UNLIKELY_EOF
   fi
   "$VENV_BIN/letsencrypt" "$@"
 
+  if [ "$configure_cron" = 1 ]; then
+    if [ -f /etc/debian_version ]; then
+     echo "Configuring cron for Debian."
+     ConfigureCronDeb
+    elif [ -f /etc/redhat-release ]; then
+      echo "Configuring cron for Redhat."
+      ConfigureCronCommon "/var/spool/cron/root"
+    elif [ -f /etc/os-release ] && `grep -q openSUSE /etc/os-release` ; then
```

Comment:
I realize this uses the exact same fingerprinting method than the one for picking correct bootstrapper, and isn't most likely in the scope of this PR, but I feel it'd be better to use values of either `ID` or `ID_LIKE` instead `PRETTY_NAME` as I think these would be more stable over releases.